libs/dh/shared/ui-util/src/lib/dh-enum-to-dropdown-options.ts

```diff
@@ -16,9 +16,24 @@
  */
 import { WattDropdownOptions } from '@energinet-datahub/watt/dropdown';
 
-export function enumToDropdownOptions<T extends object>(enumObj: T) {
-  return Object.keys(enumObj).map((key) => ({
-    displayValue: key,
-    value: Object.values(enumObj)[Object.keys(enumObj).indexOf(key)],
-  })) as WattDropdownOptions;
+type Sort = 'asc' | 'desc';
+
+export function DhEnumToWattDropdownOptions<T extends object>(
```

Comment:
nitpick: throughout the project functions start with a small letter, so let's keep that convention.